architecture/consumption-architecture-dependency-management.md

```diff
@@ -0,0 +1,148 @@
+# Introduction
+## Context and Scope
+This document captures technologies that are hosted in OpenSSF and some technologies in CNCF. The technical stack in this document demonstrates how OSS security technologies make software supply chain more secure through dependency management. 
+
+The goals of this document are:
+1. To help open source producers quickly navigate the OSS security technology landscape, discover, adopt and contribute to technical initiatives. 
+2. To provide an easy model for our end user organizations large and small to have a framework/reference architecture to help them think about adopting OpenSSF technical projects and guidance. 
```

Comment:
"think about" is weak. Maybe "consider"?

Reply:
Thank you @david-a-wheeler for taking the time. Updated the wording per your suggestion.